lib/src/facebook_login.dart

```diff
@@ -50,6 +56,17 @@ class FacebookLogin {
     return _isLoggedIn(token);
   }
 
+	/// Set Application ID.
+	Future<void> setAppId({String? appId}) {
```

Comment:
Make `appId` non nullable and required. In this implementation there is no sense to call this method without an `appId`.